retroarch.cfg

```diff
@@ -138,6 +96,18 @@
 # By default, tries to use first suitable driver.
 # video_context_driver =
```

Comment:
Is the `video_context_driver` option accessible from RGUI?

Reply:
Nope it isn't. I would leave that out for now.

Reply:
Should I move it back to `Video`, or should it go in `Misc`?